clients/nodejs/index.js

```diff
@@ -22,27 +23,42 @@ if (argv['log-tag']) {
         argv['log-tag'] = [argv['log-tag']];
     }
     argv['log-tag'].forEach((lt) => {
-        let s = lt.split(':');
+        const s = lt.split(':');
         Nimiq.Log.instance.setLoggable(s[0], s.length == 1 ? 2 : s[1]);
     });
 }
 
-console.log('Nimiq NodeJS Client starting (host=' + host + ', port=' + port + ', miner=' + !!miner + ', passive=' + !!passive + ')');
+console.log(`Nimiq NodeJS Client starting (host=${host}, port=${port}, miner=${!!miner}, passive=${!!passive})`);
 
 // XXX Configure Core.
 // TODO Create config/options object and pass to Core.get()/init().
 Nimiq.NetworkConfig.configurePeerAddress(host, port);
 Nimiq.NetworkConfig.configureSSL(key, cert);
 
-(new Nimiq.Core()).then($ => {
-    console.log('Blockchain: height=' + $.blockchain.height + ', totalWork=' + $.blockchain.totalWork + ', headHash=' + $.blockchain.headHash.toBase64());
+const options = {
+    walletSeed
```

Comment:
Since I think we don't use shorthand property names anywhere else (I think we only use shorthand method names), I would go with the explicit syntax here.